### ISSUE TITLE
Error 500: Reference data

### ISSUE THREAD

**Issue by xbaro** [codalab/codalab-competitions#1619]
Dear all, 

on the competition with id 7091 there is something wrong. If we submit a prediction the the current phase it say that there is no reference data (but in the edit page it appears as selected). 

If we try to change the data, on save it appears an error 500.

Xavier

**Comment by FlavioAlexander** [comment]
@xbaro I am having a hard time reproducing the error. Have you seen this issue before? It seems like you can't save the changes when editing your competition correct? I have looked the logs,  it shows an `internal error`, but does not give me much information to determine the reasons. Also, I checked the reference data for all your phases, they seem to exist and have the proper path. 

Best, 

Flavio

**Comment by xbaro** [comment]
Hi, we did not have the problem before. 

Something changed from our last competitions. Generally, we were only allowed to see our own datasets, while the last time we saw a large list of datasets from other persons. 

Now the list is not so long, but still I can see datasets that are not mine. 

In any case, to reproduce the error is easy, I just go to edit the competition and save (no changes are required). The error 500 is raised.
